README.md

```diff
@@ -14,9 +14,74 @@ You're in control:
+|IAMRevoke|IAM|Revokes IAM permissions granted by an anomolous grant|
+|OpenFirewall|Compute Engine|Closes an firewall rule that has 0.0.0.0/0 ingress open|
+|RemovePublicIP|Compute Engine|Removes external IP from a GCE instance|
+|SnapshotDisk|Compute Engine|Creates a disk snapshot in response to a C2 finding|
+|UpdatePassword|Cloud SQL|Updates the Cloud SQL root password|
+
+---
+
+## Configuration
+### Filter
+
+_NOTE: Filters are only supported if using SCC Notifications_
+
+Sometimes in your environment, you'll run into a scenario where a finding is a false positive because it is expected in your environment. In this case, we use the Filter Cloud Function to automatically mark findings as false positives in SCC and then set them as INACTIVE so you don't have to alert on them. To filter, we use a common policy language used in other Google Cloud open source called [Rego](https://www.openpolicyagent.org/docs/latest/#rego) from the good folks at [Open Policy Agent](https://www.openpolicyagent.org/).
+
+To add your own Rego files simply add them in `./config/filters`. The Cloud Function will pick up any files with the `.rego` extension except `*_test.rego` so please also add tests. Each file must have a single "rule" where `deny` is true (and the finding is filtered) if the block evaluates to `true`. For example, let's say in a particular project that is low risk, we want to filter out Bad IP findings that look like a valid NTP request, since many times they are. The rego would look like this:
+
+```rego
+# filename: ntpd.rego
+
+package sra.filter
+
+ntpd {
+	ipcon := input.finding.sourceProperties.properties.ipConnection
+	ipcon.destPort == 123
+	ipcon.protocol == 17
+}
+```
+
+A few notes on the syntax:
+
+1. The package name must be `sra.filter` and the rule must match the filename (without the extension) since the query the Cloud Function
+uses is `data.sra.filter.<filename-without-extension>`
+2. Rego's entry point is called "input" so since the finding JSON has a root node of "finding", we'll address it by `input.finding` and then all the nested fields beneath it.
+3. You can think of each line in a block as a boolean condition that is AND'ed together. So below we are asserting the destPort must equal 123 and the protocol must equal 17 (UDP) for this finding to be filtered out. For more on the Rego syntax see: [https://godoc.org/github.com/open-policy-agent/opa/rego](https://godoc.org/github.com/open-policy-agent/opa/rego)
+
+#### Testing Filters
+
+OPA gives you the ability to test your Rego policies against actual JSON. To do this, simply add the Notification JSON structure into the test and make assertions against it. We give an example of this in `./config/filters/false_positive_test.rego`. You can run tests yourself  after you [download OPA](https://www.openpolicyagent.org/docs/latest/#running-opa) by trying the following:
+
+```
+cp config/filters/false_positive.rego config/filters/false_positive.rego
```

Comment:
maybe this line should be 
```
cp config/filters/false_positive.rego.sample config/filters/false_positive.rego
```
@onetwopunch

Reply:
@daniel-cit good catch, would very much appreciate a full review and I can make another PR to address the findings.

Reply:
This is a huge improvement, thanks a lot for this :) 

There is only one point that I think that a small change could be useful, which is to set the default value of variable `enable-scc-notification` to `true`

1) The findings that are exported to logging are only ETD findings, but both ETD and SHA  are exported using SCC notifications 
2) ETD findings are only available in the `Premium tier` (https://cloud.google.com/security-command-center/pricing#tier-pricing) while many useful SHA findings like `OPEN_FIREWALL`, `PUBLIC_BUCKET_ACL` and `PUBLIC_SQL_INSTANCE`  are available in the `Standard tier`

I think that having some SHA findings available by default in the Standard tier could be very helpful to the all user.
@onetwopunch

Reply:
Happy to set the variable to true by default! Just wasn't sure how if that would affect current users. Since we seem to be ok with some breaking changes though that's fine with me.

Reply:
That is a good point.

Maybe a note in the README explaining the braking change that now the SCC notification is created by default but can be turned off would help so that  the users are not taken by surprise.

In the  [Google Cloud security foundations guide](https://services.google.com/fh/files/misc/google-cloud-security-foundations-guide.pdf) section 10.1.3 Setting up basic security alerting , using SCC notifications  is the recommendation, so that is way I am suggesting to set it to `true` by default .